src/app/person.ts

```diff
@@ -16,6 +16,7 @@ export class Person {
     constructor(
```

Comment:
Please make sure the copyright date range in the header of each file you have touched extends to 2020, like this:

https://github.com/google/peoplemath/blob/37da3686c7e2fdac2b77fb130233b104ca610562/src/app/objective.ts#L1

I bet there's an editor plugin or something out there that can do this for you automatically.